api/view/static/openapi.json

```diff
@@ -0,0 +1,298 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "Geometry Data Service",
+    "version": "1.0.0",
+    "title": "Geometry Data Service",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags": [
+    {
+      "name": "geometry",
+      "description": "Geometries"
+    },
+    {
+      "name": "search",
+      "description": "Perform search functions"
+    }
+  ],
+  "paths": {
+    "/geometry": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "List geometries available",
```

Comment:
If there are 2 Billion geometries in the service, does this get them all? Are they paged at least?